analysis/codelists.py

```diff
@@ -0,0 +1,36 @@
+from ehrql import codelist_from_csv
+
+acute_otitis_media_cod = codelist_from_csv(
+    "codelists/opensafely-pharmacy-first-acute-otitis-media-treatment-dmd.csv",
+    column="code",
+)
+
+first_impetigo_treatment_cod = codelist_from_csv(
+    "codelists/opensafely-pharmacy-first-impetigo-treatment-dmd.csv",
+    column="code",
+)
+
+infected_insect_bites_cod = codelist_from_csv(
+    "codelists/opensafely-pharmacy-first-infected-insect-bites-treatment-dmd.csv",
+    column="code",
+)
+
+shingles_treatment_cod = codelist_from_csv(
+    "codelists/opensafely-pharmacy-first-shingles-treatment-dmd.csv",
+    column="code",
+)
+
+sinusitis_cod = codelist_from_csv(
```

Comment:
To avoid later confusion add word treatment here (as there is also a diagnosis code for same condition).
```suggestion
sinusitis_treatment_cod = codelist_from_csv(
```